### PR TITLE
[WIP] Remove montages with set_montage(None)

### LINKED ISSUE #4841: Reset montage in raw?
I was wondering if the montage can somehow be reset in an existing `raw` object. Suppose I did `raw.set_montage('biosemi64')`, but later I decide I want to reset the montage to nothing - how can I do that?

Does this make sense (during interactive data exploration it would be useful for me)? If yes, is it doable (I assume only if `update_ch_names=False`, because otherwise how should we reconstruct the original channel names)?

### PR DESCRIPTION
**Pull request by cbrnr**
Fixes #4841.

Questions:

1. Do we care about the `set_dig` argument?
2. I'm not sure what the correct default value for `info["chs"][idx]["loc"]` is. `mne.create_info` fills this entry with `np.concatenate((np.zeros(3), np.eye(3).ravel())).astype(float)`, whereas `mne.io.read_raw_edf` seems to fill it with all zeros.

### CODE REVIEW

**Comment by larsoner** [comment]
As far as `set_dig` goes, yes if it is `True` then you should wipe `info['dig']` as well as `chs[ii]['loc']`.

For the `loc` it won't make too much difference which you choose. Let's things simple and change `create_info` to use all zeroes to make it clear that it's not really defined.

**Comment by cbrnr** [comment]
> As far as `set_dig` goes, yes if it is `True` then you should wipe `info['dig']` as well as `chs[ii]['loc']`.

OK, let me make sure I understand this correctly. I always wipe `info["chs"][idx]["loc"]` (i.e. set it to a zero array), and in addition I wipe `info["dig"]` (i.e. set it to `None`) only if `set_dig=True`. Correct?

I'll change `create_info` to use all zeros then.

**Comment by larsoner** [comment]
Sounds right to me

File: mne/io/meas_info.py [hunk 1]
```diff
@@ -1723,7 +1723,7 @@ def create_info(ch_names, sfreq, ch_types=None, montage=None, verbose=None):
                          '(%s != %s)' % (len(ch_types), nchan))
     info = _empty_info(sfreq)
     info['meas_date'] = np.array([0, 0], np.int32)
-    loc = np.concatenate((np.zeros(3), np.eye(3).ravel())).astype(np.float32)
+    loc = np.zeros(12)
```
Comment (agramfort) [inline mne/io/meas_info.py]: is that a bug fix?

Comment (cbrnr) [reply mne/io/meas_info.py]: @larsoner suggested to keep things simple and initialize with all zeroes (see his comment above).

**Comment by agramfort** [comment]
if it does not break anything why not.

**Comment by cbrnr** [comment]
I simplified this further (please check). If CIs are happy this is ready to be merged.

**Comment by larsoner** [comment]
CIs were happy before I fixed the conflict, merging -- thanks @cbrnr

**Pull request merged**